### ISSUE TITLE
MIssing `fonts` folder in `skins/ui/oxide-dark`

### ISSUE THREAD

**Issue by skrtheboss** [tinymce/tinymce#5048]
**What is the current behavior? Describe the bug**
The file `skins/ui/oxide-dark/skin.mobile.css` has a reference to `fonts/tinymce-mobile.woff` but this folder is missing.

![TinyMce-oxide-dark-folder-structure](https://user-images.githubusercontent.com/10209728/59927699-5d417e80-943d-11e9-8317-5832dada0a9b.PNG)

`oxide-dark/skin.mobile.css`
```css
@font-face {
  font-family: 'tinymce-mobile';
  font-style: normal;
  font-weight: normal;
  src: url('fonts/tinymce-mobile.woff?8x92w3') format('woff');
}
```

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

**What is the expected behavior?**
It should behave just like the `oxide` skin.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
`v

5.0.8`

**Comment by TheSpyder** [comment]
Ah. See how the font file is duplicated in the `oxide` and `oxide-dark` folder? We migrated the font file recently and the build process was still using `flatten: true` when copying the files. The copy in `oxide/fonts` was left over from the previous build process.

Oops.

**Comment by lnewson** [comment]
This issue has been fixed in TinyMCE 5.0.10, which has just been released to the community. Please let us know if you have any further issues.
